rules/linux/network_connection/net_connection_lnx_ngrok_tunnel.yml

```
@@ -0,0 +1,26 @@
+title: Communication To Ngrok Tunneling Service
```

I think adding the `- Linux` at the end is better to distinguish it from the windows one. Since they both have the same name

```suggestion
title: Communication To Ngrok Tunneling Service - Linux
```